skellytracker/trackers/openpose_tracker/openpose_tracker.py

```diff
@@ -0,0 +1,129 @@
+import subprocess
+from pathlib import Path
+from typing import Union
+from skellytracker.trackers.base_tracker.base_tracker import BaseCumulativeTracker
+from skellytracker.trackers.openpose_tracker.openpose_recorder import OpenPoseRecorder
+
+import os
+
+
+class OpenPoseTracker(BaseCumulativeTracker):
+    def __init__(
+        self,
+        openpose_exe_path: Union[str, Path],
+        output_json_path: Union[str, Path],
+        net_resolution: str = "-1x640",
+        number_people_max: int = 1,
+    ):
+        """
+        Initialize the OpenPoseTracker.
+
+        :param recorder: An instance of OpenPoseRecorder for handling the output.
+        :param openpose_exe_path: Path to the OpenPose executable.
+        :param output_json_path: Directory where JSON files will be saved.
+        :param net_resolution: Network resolution for OpenPose processing.
+        :param number_people_max: Maximum number of people to detect.
+        """
+        # super().__init__(recorder=recorder)
```

Comment:
Is there a reason we're not initializing the super here?

Reply:
I genuinely don't remember - but I think it's because a few lines down (line 33-36), I initialized the recorder there, and probably forgot to delete the commented out version in line 27.